src/concepts/essentials/cli_vs_desktop.md

```diff
@@ -1,13 +1,13 @@
 # CLI vs Desktop
```

Comment:
Thanks I suck at english :D

Reply:
Every bilingual person says that while us Americans can't even speak our primary language properly hahaha.